packages/solid/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@lottiefiles/dotlottie-solid",
+  "version": "0.0.0",
+  "type": "module",
+  "description": "Solid wrapper around the dotlottie-web library",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LottieFiles/dotlottie-web.git",
+    "directory": "packages/solid"
+  },
+  "homepage": "https://github.com/LottieFiles/dotlottie-web#readme",
+  "bugs": "https://github.com/LottieFiles/dotlottie-web/issues",
+  "author": "LottieFiles",
+  "contributors": [],
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "exports": {
+    "development": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/dev.js"
+      }
+    },
+    "import": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    }
+  },
+  "browser": {},
+  "types": "./dist/index.d.ts",
+  "typesVersions": {},
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "solid",
+    "dotlottie",
+    "lottie",
+    "player",
+    "animation",
+    "web",
+    "canvas",
+    "javascript",
+    "thorvg"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "dev": "vite serve dev",
+    "lint": "eslint --fix .",
+    "stats:eslint": "cross-env TIMING=1 eslint .",
+    "stats:ts": "tsc -p tsconfig.build.json --extendedDiagnostics",
+    "type-check": "tsc --noEmit"
+  },
+  "peerDependencies": {
+    "solid-js": "^1.6.12"
+  },
+  "dependencies": {
+    "@lottiefiles/dotlottie-web": "workspace:*",
+    "debounce": "^2.0.0"
+  },
+  "devDependencies": {
+    "cross-env": "7.0.3",
+    "solid-js": "^1.8.17",
+    "tsup": "^8.0.2",
+    "tsup-preset-solid": "^2.2.0",
+    "typescript": "^5.4.5",
+    "vite": "^5.2.12",
+    "vite-plugin-solid": "^2.10.2"
```

Comment:
I don't think we need these vite related dependencies anymore

Reply:
Resolved on https://github.com/LottieFiles/dotlottie-web/pull/234/commits/79677f98bb727a8ee983f9c04c12b6267943593d